package.json

```diff
@@ -22,6 +22,7 @@
     "classnames": "2.2.5",
     "css-element-queries": "0.4.0",
     "d3": "4.5.0",
+    "guid": "0.0.12",
```

Comment:
If you are adding new package to quick-react-ts, you have to rebuild npm shrinkwrap file